arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -182,5 +182,44 @@ public async Task TruncateCollectionAsync_ShouldThrow_WhenCollectionDoesNotExist
 
             Assert.Equal(ex.ApiError.ErrorNum, 1203);
         }
+
+        [Fact]
+        public async Task GetCollectionsAsync_ShouldSucceed()
+        {            
+            string[] collectionNames = new string[] { "MyFirstCollection", "MySecondCollection", "MyThirdCollection" };
+            await GenerateCollections(collectionNames);
+
+            var response = await _collectionApi.GetCollectionsAsync(new GetCollectionsOptions
+            {
+                ExcludeSystem = true // System adds 9 collections that we don't need to test
+            });
+            Assert.NotEmpty(response.Result);
+            var collectionExists = response.Result.Where(x => x.Name == "MyFirstCollection");
+
+            Assert.False(response.Error);
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.NotNull(collectionExists);
+            await DisposeCollections(collectionNames);
```

Comment:
I'm not sure it's ideal to do this - if the test fails for any reason then the collections are not removed.  

For performance it might be better to create all the collections we need for the tests once in the Fixture class.  The fixture class already takes care of deleting the whole database so we also wouldn't have to worry about removing them.